spec/features/food_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Food', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Name', email: 'e@g.com', password: 123_456, id: 1)
+    @food = Food.create(name: 'f_name', measurement_unit: 'kg', quantity: 30, price: 10, user: @user)
```

Comment:
- [ ] Basically, your unit tests here are failing because you are not creating the food. If you look keenly in your schema.rb file, the column for user is supposed to be ```user_id``` but here you are having ```user``` note the difference? When you change this the tests are all passing correctly and the 4 failing tests pass.

**The errors before changing the user to user_id**

![image](https://user-images.githubusercontent.com/86318284/202519250-6bd093a4-7bd6-4b44-b2c4-d498fab15181.png)

**The Unit tests now passing after changing the user to user_id**

![image](https://user-images.githubusercontent.com/86318284/202519470-a957ec6d-62a0-4965-b9e0-d938f7111d8d.png)